components/host-dashboard/AddFundsModal.js

```diff
@@ -170,6 +170,8 @@ const getOptions = (amount, currency, intl) => {
 
 const AddFundsModal = ({ host, collective, ...props }) => {
   const { LoggedInUser } = useUser();
+  const [platformTipError, setPlatformTipError] = useState(null);
```

Comment:
What's the benefit of using our own state here? Using the `error` from `useMutation` simplify things as we don't have to set (and reset) the state ourselves

Reply:
@Betree : The only reason I did this way is because I couldn't find a simple way of resetting the error on modal close. So for example if we use the error from useMutation, then when the modal closes how do we reset it back? It seems that it's not auto reset when the modal closes and I couldn't figure out exactly why that's the case. 🤔

Reply:
@Betree : Ops, I am mistaken, somehow my previous tests didn't work, and maybe I did something wrong and it wasn't somehow resetting upon model closure; but when I go for a walk and came back and tried this again now it's working perfectly. 😄 😄